x-pack/plugins/session_view/public/components/session_view/styles.ts

```diff
@@ -34,6 +34,11 @@ export const useStyles = ({ height = 500 }: StylesDeps) => {
       zIndex: 2,
     };
 
+    const nonGrowGroup: CSSObject = {
```

Comment:
this can be removed now

Reply:
@opauloh can you help us remove this in a follow up PR or coordinate with @kqualters-elastic on removing this? Thanks!